packages/@okta/vuepress-site/docs/reference/api/myaccount/index.md

```diff
@@ -302,6 +610,73 @@ curl -XPUT 'https://${yourOktaDomain}/api/v1/myaccount/directoryProfile' -H 'Aut
 
 ## MyAccount API objects
 
+### My Email object
+
+#### My Email properties
+
+| Property           | Type                                                            | Description                                                                                                       |
+| ------------------ | --------------------------------------------------------------- | ------------------------------------------------------ |
+| `id`               | String                                                            | The caller's email ID|
+| `status`           | String                                                            | The caller's email status, VERIFIED, OR UNVERIFIED|
+| `roles`            | Object                                                            | The roles object defines the role of the email, PRIMARY or SECONDARY|
+| `profile`          | Object                                                            | The profile object defines the email address on the profile.|
+| `_links`           | Object ([JSON HAL](http://tools.ietf.org/html/draft-kelly-json-hal-06))  | Discoverable resources related to the caller's email |
+
+#### My Email example
+
+```json
+{
+    "id": "5a8de6071e1b94e0f4ec664b9e4869e8",
+    "status": "VERIFIED",
+    "profile": {
+        "email": "some.primary.email@okta.com"
+    },
+    "roles": [
+        "PRIMARY"
+    ],
+    "_links": {
+        "self": {
+            "href": "https://${yourOktaDomain}/idp/myaccount/emails/5a8de6071e1b94e0f4ec664b9e4869e8",
+            "hints": {
+                "allow": [
+                    "GET"
+                ]
+            }
+        },
+        "challenge": {
+            "href": "https://${yourOktaDomain}/idp/myaccount/emails/5a8de6071e1b94e0f4ec664b9e4869e8/challenge",
+            "hints": {
+                "allow": [
+                    "POST"
+                ]
+            }
+        }
+    }
+}
+```
+
+### My Email Request object
+
+#### My Email Request properties
+
+| Property           | Type                                                            | Description                                                                                                       |
+| ------------------ | --------------------------------------------------------------- | ------------------------------------------------------ |
+| `profile`          | Object                                                            | The profile object defines the email address on the profile.|
+| `roles`            | Object                                                            | The roles object defines the role of the email, PRIMARY or SECONDARY|
```

Comment:
This should be `role` according to `IdpMyAccountEmailRequest`. Besides I have created OKTA-478482 to update the schema because this field is missing.

Reply:
made the change ty